host-configs/llnl/toss_3_x86_64_ib/clang@10.0.1.cmake

```diff
@@ -10,11 +10,11 @@
 # This file provides CMake with paths / details for:
 #  C,C++, & Fortran compilers + MPI
 #------------------------------------------------------------------------------
-# clang-4.0.0 / gfortran@4.9.3 compilers
+# clang-10.0.1 / gfortran@8.3.1 compilers
 # Uses clang's 'libc++' instead of 'libstdc++'
```

Comment:
```suggestion
# Uses gcc@8.3.1 standard library'
```

I think its more common to link into the gcc 8.3.1 standard library than use the clang one.

Because it is out of scope for a suggestion, change the whole extra flags section to just be this:

https://github.com/LLNL/serac/blob/6659a35e753f14ff39eaab48a0b3f89e4d5e9526/host-configs/quartz-toss_3_x86_64_ib-clang%4010.0.0.cmake#L30-L32